build/go.mod

```diff
@@ -0,0 +1,12 @@
+module github.com/mattermost/mattermost-plugin-starter-template/build
+
+go 1.12
+
+require (
+	github.com/mattermost/mattermost-server/v5 v5.39.0
```

Comment:
Why is this imported? Is there code still using it?

Reply:
`build/deploy/main.go` still used v5, upgraded the dep to v6 and tested :-)